src/fidesops/ops/tasks/__init__.py

```diff
@@ -20,6 +35,8 @@ def _create_celery(config_path: str = config.execution.celery_config_path) -> Ce
         # Defaults for the celery config
         "broker_url": config.redis.connection_url,
         "result_backend": config.redis.connection_url,
+        # Fidesops requires this to route emails to separate queues
+        "task_create_missing_queues": True,
```

Comment:
This var. defaults to `True` anyway, and is able to be overridden here, but I've added this for transparency